src/engraving/libmscore/transpose.cpp

```diff
@@ -429,6 +429,9 @@ bool Score::transpose(TransposeMode mode, TransposeDirection direction, Key trKe
     }
 
     Segment* s1 = _selection.startSegment();
+    if (!s1) {
+        return result;
```

Comment:
nice catch!
also I've noticed in flie "scorefile.cpp" that startSegment isn't checked, maybe should be added?
`measureStart = _selection.startSegment()->measure();`